From: Maximilian Weigand <mweigand@mweigand.net>
Date: Wed, 22 Nov 2023 16:46:16 +0100
Subject: pinenote-dts: based on recommendation by user Jazn,
 increase overvoltage value of the backlight to 32 V (reduces spikes on the
 power lines)

---
 arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
index 7668124..bbd68f46 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-pinenote.dtsi
@@ -586,8 +586,8 @@ led-controller@36 {
 		ti,boost-frequency-hz = <1120000>;
 
 		/* ti,boost-ovp-microvolt = <16000000>; */
-		ti,boost-ovp-microvolt = <24000000>;
-		/* ti,boost-ovp-microvolt = <32000000>; */
+		/* ti,boost-ovp-microvolt = <24000000>; */
+		ti,boost-ovp-microvolt = <32000000>;
 		/* ti,boost-ovp-microvolt = <40000000>; */
 
 		ti,boost-ocp-microamp = <600000>;
